overlord/overlord.go

```diff
@@ -33,10 +37,14 @@ import (
 	"github.com/ubuntu-core/snappy/overlord/state"
 )
 
+var ensureInterval = 3 * time.Second
```

Comment:
I wonder how we're planning on doing this in the end.

Are we going to have the device wake up every three seconds?
Is this just a temporary development feature that we plan to replace later?
Are we going to keep it but make the interval much larger (~ hours)